packages/components/molecules/f-navigation-links/src/components/NavigationLinks.vue

```diff
@@ -1,32 +1,79 @@
 <template>
-    <div
+    <ul
+        v-if="links.length"
         :class="$style['c-navigationLinks']"
         data-test-id="navigationLinks">
-        I am a NavigationLinks Component (GB)
-    </div>
+        <li
+            v-for="({ id, url, name, selected }, i) in links"
+            :key="i"
+            :class="$style['c-navigationLinks-item']">
+            <span
+                v-if="selected"
+                :data-test-id="id"
+                tabindex="0"
+                :class="[$style['c-navigationLinks-link'], $style['c-navigationLinks--active']]">
```

Comment:
Missed the naming change to the second class:

```suggestion
                :class="[$style['c-navigationLinks-link'], $style['c-navigationLinks-link--active']]">
```